docs/streams.html

```diff
@@ -398,6 +401,85 @@
         in <code>StreamsConfig</code> before running it. A complete list of
         Kafka Streams configs can be found <a href="#streamsconfigs"><b>here</b></a>.
         </p>
+
+        <h2><a id="streams_upgrade" href="#upgrade">Upgrade guide and API changes</a></h2>
+
+        <h3><a id="streams_upgrade_1020" href="#upgrade_1020">Upgrading a Kafka Streams Application</a></h3>
+
+
+        <h5>Upgrading 0.10.1.x to 0.10.2.0</h5>
+        <p>
+        See <a href="../#upgrade_1020_streams">Upgrade Section</a> for details.
+        </p>
+
+        <h3><a id="streams_api_changes" href="#api_changes">Streams API changes in 0.10.2.0</a></h3>
+        <li> New methods in <code>KafkaStreams</code>:
+            <ul>
+                <li> set a listener to react on application state change via <code>#setStateListener(StateListener listener)</code> </li>
+                <li> retrieve the current application state via <code>#state()</code> </li>
+                <li> retrieve the global metrics registry via <code>#metrics()</code> </li>
+                <li> apply a timeout when closing an application via <code>#close(long timeout, TimeUnit timeUnit)</code> </li>
+                <li> specify a custom indent when retrieving Kafka Streams information via <code>#toString(String indent)</code> </li>
+            </ul>
+        </li>
+        <li> Parameter updates in <code>StreamsConfig</code>:
+            <ul>
+                <li> parameter "zookeeper.connect" got deprecated </li>
+                <li> added many new parameters for metrics, security, and client configurations </li>
```

Comment:
I am ok with not listing them all here, but it would help if we could point the reader to a place where all the parameters are listed (not necessarily a list that highlights specifically the new ones, just the full list of currently supported ones).